app/controllers/job_invocations_controller.rb

```diff
@@ -58,6 +58,8 @@ def show
                                      .where(:template_invocations => { :job_invocation_id => @job_invocation.id})
     end
     @hosts = resource_base_search_and_page
+    @job_organization = Taxonomy.find_by(id: @job_invocation.task.input[:current_organization_id])
+    @job_location = Taxonomy.find_by(id: @job_invocation.task.input[:current_location_id])
```

Comment:
How about
```ruby
@job_organization = Organization.find(@job_invocation.task.input[:current_organization_id])
@job_location = Location.find(@job_invocation.task.input[:current_location_id])
```

Reply:
Ah, that behaves a bit differently when the record cannot be found, thanks for reminding me.